app/renderer/js/preload.ts

```diff
@@ -1,51 +1,55 @@
 'use strict';
 
-const { ipcRenderer, shell } = require('electron');
-const SetupSpellChecker = require('./spellchecker');
+import { ipcRenderer, shell } from 'electron';
+import SetupSpellChecker from './spellchecker';
 
-const ConfigUtil = require(__dirname + '/utils/config-util.js');
-const LinkUtil = require(__dirname + '/utils/link-util.js');
-const params = require(__dirname + '/utils/params-util.js');
+import ConfigUtil = require('./utils/config-util');
+import LinkUtil = require('./utils/link-util');
+import params = require('./utils/params-util');
 
 // eslint-disable-next-line import/no-unassigned-import
 require('./notification');
 
 // Prevent drag and drop event in main process which prevents remote code executaion
 require(__dirname + '/shared/preventdrag.js');
 
-// eslint-disable-next-line camelcase
+declare let window: MyWindow;
+
+// eslint-disable-next-line @typescript-eslint/camelcase
 window.electron_bridge = require('./electron-bridge');
 
-const logout = () => {
+const logout = (): void => {
 	// Create the menu for the below
-	document.querySelector('.dropdown-toggle').click();
+	const dropdown = document.querySelector('.dropdown-toggle');
+	(dropdown as HTMLElement).click();
 
 	const nodes = document.querySelectorAll('.dropdown-menu li:last-child a');
-	nodes[nodes.length - 1].click();
+	(nodes[nodes.length - 1] as HTMLElement).click();
```

Comment:
Minor suggestion, you can use do `HTMLAnchorElement` here. Also I think we can also do `NodeList<HTMLAnchorArray>[]` or similar on `nodes variable` if you can figure this out.